addons/editor-theme3/color_on_white.css

```
@@ -66,6 +66,11 @@
   color: #575e75;
 }
 
+.blocklyCommentTopbar > image,
+.scratchCommentTopBar ~ image {
+  filter: brightness(0);
```

```suggestion
  filter: none;
```
This will make the color of the icons the same as the text.